docs/platforms/javascript/common/user-feedback/configuration/index.mdx

```diff
@@ -250,10 +250,10 @@ function MyFeedbackButton() {
   }
 
   return (
-    <button type="button" onClick={() => {
+    <button type="button" onClick={async () => {
       const form = await feedback.createForm();
       form.appendToDom();
-      form.show();
+      form.open();
```

Comment:
`open` is the func to call, per https://github.com/getsentry/sentry-javascript/blob/531779300c186f89afff0c5bad9f802b2140a325/packages/types/src/feedback/index.ts#L44-L52